crates/ev-cli/src/theme.rs

```diff
@@ -0,0 +1,373 @@
+                unpicked_item_prefix: style(" ".to_string()).for_stderr(),
+                inline_selections: true,
+            }
+        } else {
+            CliTheme {
+                defaults_style: Style::new().for_stderr().cyan(),
+                prompt_style: Style::new().for_stderr().green().bold(),
+                prompt_prefix: style("?".to_string()).for_stderr().green().dim(),
+                prompt_suffix: style("›".to_string()).for_stderr().black().bright(),
+                success_prefix: style("✔".to_string()).for_stderr().green().dim(),
+                success_suffix: style("·".to_string()).for_stderr().black().bright(),
+                error_prefix: style("✘".to_string()).for_stderr().red(),
+                error_style: Style::new().for_stderr().red(),
+                hint_style: Style::new().for_stderr().black().bright(),
+                values_style: Style::new().for_stderr().cyan().dim(),
+                active_item_style: Style::new().for_stderr().bold(),
+                inactive_item_style: Style::new().for_stderr(),
+                active_item_prefix: style("❯".to_string()).for_stderr().bold(),
+                inactive_item_prefix: style(" ".to_string()).for_stderr(),
+                checked_item_prefix: style("✔".to_string()).for_stderr().green(),
+                unchecked_item_prefix: style("✔".to_string()).for_stderr().black(),
+                picked_item_prefix: style("❯".to_string()).for_stderr().green(),
+                unpicked_item_prefix: style(" ".to_string()).for_stderr(),
+                inline_selections: true,
+            }
+        }
+    }
+}
+
+impl Theme for CliTheme {
+    /// Formats a prompt.
+    fn format_prompt(&self, f: &mut dyn fmt::Write, prompt: &str) -> fmt::Result {
+        if !prompt.is_empty() {
+            write!(
+                f,
+                "{} {} ",
+                &self.prompt_prefix,
+                self.prompt_style.apply_to(prompt)
+            )?;
+        }
+
+        write!(f, "{}", &self.prompt_suffix)
+    }
+
+    /// Formats an error
+    fn format_error(&self, f: &mut dyn fmt::Write, err: &str) -> fmt::Result {
+        write!(
+            f,
+            "{} {}",
+            &self.error_prefix,
+            self.error_style.apply_to(err)
+        )
+    }
+
+    /// Formats an input prompt.
+    fn format_input_prompt(
+        &self,
+        f: &mut dyn fmt::Write,
+        prompt: &str,
+        default: Option<&str>,
+    ) -> fmt::Result {
+        if !prompt.is_empty() {
+            write!(
+                f,
+                "{} {} ",
+                &self.prompt_prefix,
+                self.prompt_style.apply_to(prompt)
+            )?;
+        }
+
+        match default {
+            Some(default) => write!(
+                f,
+                "{} {} ",
+                self.hint_style.apply_to(&format!("({})", default)),
+                &self.prompt_suffix
+            ),
```

Comment:
default is an [unstable keyword in rust](https://github.com/rust-lang/rust/issues/31844), should avoid using in vars:
```suggestion
            Some(default_val) => write!(
                f,
                "{} {} ",
                self.hint_style.apply_to(&format!("({})", default_val)),
                &self.prompt_suffix
            ),
```

Reply:
til